exercises/concept/lasagna-master/.docs/introduction.md

```diff
@@ -95,7 +95,7 @@ function divide(a, b) {
 ## Function Expression
 
 A function declaration is a standalone statement.
-But sometimes it is helpful to define a function as part of another expression, e.g., in an assignment, as a function parameter ([callback][concept-callbacks]) or as value in an [object][concept-objects].
+But sometimes it is helpful to define a function as part of another expression, e.g. in an assignment, as a function parameter ([callback][concept-callbacks]) or as value in an [object][concept-objects].
```

Comment:
I am ok with merging this change but please take note of the Exercism guidelines regarding this:
> Many American English style guides state that the abbreviations "i.e." and "e.g." should be followed by a comma [...]. This is permitted, but not required within text on Exercism.

Source: https://exercism.org/docs/building/markdown/style-guide#h-oxford-comma